### PR TITLE
Blog post for Bazel 3.6

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

**Comment by aiuto** [comment]
cc: @joehickscc    should be reviewer

**Comment by laurentlb** [comment]
The release is planned for tomorrow (Tuesday).

**Review by aiuto — APPROVED** [review]

File: _posts/2020-10-06-bazel-3-6.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Bazel 3.6"
+authors:
+  - laurentlb
+---
+
+[Bazel 3.6](https://github.com/bazelbuild/bazel/releases/tag/3.6.0) is a [minor
+release](https://docs.bazel.build/versions/3.5.0/backward-compatibility.html).
```
Comment (floriographygoth) [inline _posts/2020-10-06-bazel-3-6.md:9]: Should this be 3.5.0 or master?

File: _posts/2020-10-06-bazel-3-6.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    file. For more details, see the discussion in [issue
+    #11940](https://github.com/bazelbuild/bazel/issues/11940).
+*   The flag `--experimental_ui_limit_console_output` is removed. If you
+    use `--experimental_ui_limit_console_output=1` to silence terminal
+    output, you should use `--ui_event_filters=` instead.
+*   The flag `--proto:instantiation_stack` must be enabled with
+    `--record_rule_instantiation_callstack` to see call stack in proto output
+    from Bazel query.
+*   A few other flags are removed:
+    `--experimental_ignore_deprecated_instrumentation_spec`,
```
Comment (floriographygoth) [inline _posts/2020-10-06-bazel-3-6.md:26]: Sub bulleted list or add "and" before the last item.

File: _posts/2020-10-06-bazel-3-6.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    support them.
+*   Added support for a `supports-graceful-termination` execution requirement
+    and tag, which causes Bazel to send a SIGTERM to any tagged actions before
+    sending a delayed SIGKILL. This is to give actions, and more specifically
+    tests, a chance to clean up after themselves.
+
+
+## Other changes
+
+*   Bazel will now skip printing action stdout/stderr contents if they exceed
```
Comment (floriographygoth) [inline _posts/2020-10-06-bazel-3-6.md:81]: now skips

**Review by floriographygoth — COMMENTED** [review]
Small editorial changes, but otherwise LGTM.

File: _posts/2020-10-06-bazel-3-6.md (hunk 1 above)
Comment (laurentlb) [reply _posts/2020-10-06-bazel-3-6.md:9]: Updated

File: _posts/2020-10-06-bazel-3-6.md (hunk 2 above)
Comment (laurentlb) [reply _posts/2020-10-06-bazel-3-6.md:26]: Done

File: _posts/2020-10-06-bazel-3-6.md (hunk 3 above)
Comment (laurentlb) [reply _posts/2020-10-06-bazel-3-6.md:81]: Done, thanks.
For consistency, I've also removed a future above ("will no longer be used").

**Pull request merged**